test/unit/config/matchers/to-be-positioned-popover.js

```diff
@@ -8,7 +8,7 @@ function toBePositionedPopover( element ) {
 	const pass = element.style.top !== '' && element.style.left !== '';
 	return {
 		pass,
-		message: `Received element is ${ pass ? '' : 'not ' } positioned`,
+		message: () => `Received element is ${ pass ? '' : ' not' }positioned`,
```

Comment:
```suggestion
		message: () => `Received element is ${ pass ? '' : 'not ' }positioned`,
```

Otherwise, it will out `Received element is  notpositioned` when `pass` is false.

Reply:
Oh I thought I committed that. Good catch, thanks!